compiler/rustc_ast_lowering/messages.ftl

```diff
@@ -99,7 +99,7 @@ ast_lowering_misplaced_double_dot =
     .note = only allowed in tuple, tuple struct, and slice patterns
 
 ast_lowering_misplaced_impl_trait =
-    `impl Trait` only allowed in function and inherent method return types, not in {$position}
+    `impl Trait` only allowed in function and inherent method argument and return types, not in {$position}
```

Comment:
this will soon be out of date with RPITIT stabilization, and is also not true w.r.t TAITs. I think at some point in the future we will want to just say "not allowed in <position>" and link to the impl trait section of the reference, but for now this is fine.